experiments/optimizer/GPT2/constants.py

```diff
@@ -0,0 +1,4 @@
+model_name = "flax-community/gpt2-base-thai"
```

Comment:
Constants should be uppercase

Reply:
Done

Reply:
i am sorry for my careless.... I didnt change uppercase in run.py .... for now its already solved.